### PR TITLE
Render plc courses

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

**Comment by mehalshah** [comment]
image

**Comment by aoby** [comment]
nit: I prefer when the main screenshots are in the PR description so they appear at the top instead of a later comment. Note you can edit the description. No need to update this, just FYI for future. In any case, thanks for including screenshots.

File: dashboard/test/models/plc/user_course_enrollment_test.rb [hunk 1]
```diff
@@ -38,4 +38,28 @@ class Plc::UserCourseEnrollmentTest < ActiveSupport::TestCase
...
+      courseUnits: [
+        {
+          unitName: @course_unit1.unit_name,
+          link: Rails.application.routes.url_helpers.script_path(@course_unit1.script),
+          moduleAssignments: [],
+          status: 'start_blocked'
+        },
+        {
+          unitName: @course_unit2.unit_name,
+          link: Rails.application.routes.url_helpers.script_path(@course_unit2.script),
```
Comment (aoby) [inline dashboard/test/models/plc/user_course_enrollment_test.rb]: This should assert the expected route more explicitly, e.g. `"/s/#{@course_unit2.script.name}"`. Same above with `course_path`

**Comment by aoby** [comment]
Make sure you address the failing tests in `Plc::EnrollmentUnitAssignmentTest`

**Comment by mehalshah** [comment]
Will address tests.

My plan for PLC is to ditch the existing HAML view and replace it with an entry point that renders this control.

**Pull request merged**